protocol/minimum-base-fee.md

```diff
@@ -0,0 +1,99 @@
+
+The current prevailing base fee on Base today is slightly below 0.001 gwei. It takes 411 blocks (13.7 minutes) for the base fee to increase from 1 wei to 0.001 gwei if every block is completely full at the gas limit. (In practice, we see 70% full blocks in these scenarios, so the fees take even longer to increase.) If we could configure a minimum base fee of 0.0001 gwei, we could speed up the fee adjustment (and the corresponding end of the priority fee auction) to 59 blocks, or a little under two minutes. We could even eliminate the priority fee auctions that occur while the chain is adjusting from the minimum to the typical market fee price by setting the minimum to that price to begin with. [Arbitrum One sets a minimum of 0.01 gwei](https://docs.arbitrum.io/how-arbitrum-works/gas-fees#child-chain-gas-pricing). At recent ETH prices around $2500, any of these three minimum base fees (which correspond to `2**20` through `2**24`) would be much less than one cent for a USDC transfer (about 40,500 gas). As ETH prices change, rollup operators will want to adjust the minimum base fee accordingly.
+
+| log2(minBaseFee) | Minimum Base Fee (gwei) | USDC Transfer Cost (ETH) | Cost (USD, ~$2,500/ETH) |
+| ---------------- | ----------------------- | ------------------------ | ----------------------- |
+| 19 | 0.000524288 | 2.12e-8 | $0.0000531 |
+| 20 | 0.001048576 | 4.25e-8 | $0.000106 |
+| 21 | 0.002097152 | 8.49e-8 | $0.000212 |
+| 22 | 0.004194304 | 1.70e-7 | $0.000425 |
+| 23 | 0.008388608 | 3.40e-7 | $0.000849 |
+| 24 | 0.016777216 | 6.79e-7 | $0.00170 |
+| 25 | 0.033554432 | 1.36e-6 | $0.00340 |
+| 26 | 0.067108864 | 2.72e-6 | $0.00679 |
+| 27 | 0.134217728 | 5.44e-6 | $0.0136 |
+
+# Alternatives Considered
+
+* **Hard-code a minimum base fee** (like [EIP 7762](https://eips.ethereum.org/EIPS/eip-7762)). This is the simplest solution, but it doesn't give the chain operator any flexibility to adjust the minimum base fee. Establishing a Superchain-wide consensus on a minimum base fee seems difficult, especially if the value will have to wait for the next fork to be changed.
+* **Link the minimum base fee to an outside price signal** (like [EIP 7918](https://eips.ethereum.org/EIPS/eip-7918)). This approach avoids choosing a magic number for the minimum base fee and has more elegant economics: the fee doesn't drop when we have a good signal that reducing the base fee wouldn't increase gas usage. The downside is that it's harder to reason about the right values to use in the formula, which makes it hard to build a consensus around. A configurable minimum base fee gives us an option that L1 doesn't have: it allows rollup operators to change the minimum base fee as needed to ensure the fee market is working properly.
```

Comment:
IIUC these two EIPs apply to `blobBaseFee`, and not regular `baseFee`. Are there EIPs for adding a minimum base fee?

In general I'm interested in how we might justify adopting a different approach from L1 in addressing what seems like a common problem. I think you hint at some of the reasons above, but would be great to concentrate that argument in it's own section. In general we try to maintain as small as possible diff to L1 (of course certain things demand a diff and that's fine).